python/mosaic/api/gdal.py

```diff
@@ -66,9 +84,9 @@ def enable_gdal(spark: SparkSession) -> None:
             "GDAL not enabled. Mosaic with GDAL requires that GDAL be installed on the cluster.\n"
         )
         print(
-            "Please run setup_gdal() to generate the init script for install GDAL install.\n"
+            "You can run setup_gdal() or setup_fuse_install() to generate the init script for install GDAL install.\n"
```

Comment:
@mjohns-databricks is the name `setup_fuse_install` used instead of `setup_fuse_install_gdal` because mosaic jars are also copied? `setup_fuse_install` sounds more generic

Reply:
setup_fuse_install is for with or without GDAL (a variety of options). Since UC + Volumes have special requirements, users will use setup_fuse_install function irregardless of their need for raster processing. That is why it is now in fuse.py.